2D Array of Strings - MaxLength calculation & VowelStart string count

chinmayjain08 opened the pull request:
steps performed in the assignment: 
1. taking string input in 2D array
2. printing in matrix form
3. counting names strating with vowels
4. displaying the longest name in array

![image](https://github.com/user-attachments/assets/a6d13101-6395-41d4-a436-56a8af99e43e)

chinmayjain08 committed:
Add files via upload

chandan2853 commented on assignment-2DNamesArray.c:
```diff
@@ -0,0 +1,63 @@
+#include <stdio.h>
```
input validation and why everything is in main function?

chinmayjain08 replied:
input validation handled and function created for operations needed on matrix

chinmayjain08 committed:
input-validation and modularised

created a separate function to do all operations required in problem moving the code out of main block
